### ISSUE TITLE
perl lookalike distributions should be added to rogue distributions

### ISSUE THREAD

**Issue by oalders** [metacpan/metacpan-api#929]
```
authors/id/A/AE/AEPAGE/perl5.00502Tk800.011-win32-586.zip
authors/id/A/AN/ANDYD/perl5.002b1h.tar.gz
authors/id/B/BM/BMIDD/perl5.004_02-AlphaNTPreComp.tar.gz
authors/id/B/BM/BMIDD/perl5.00402-bindist04-msvcAlpha.tar.gz
authors/id/B/BM/BMIDD/perl5.00402-bindist05-msvcAlpha.tar.gz
authors/id/G/GR/GRABZIT/perl.2.3.0.zip
authors/id/G/GS/GSAR/perl5.00401-bindist02-bc.tar.gz
authors/id/G/GS/GSAR/perl5.00401-bindist-bc.tar.gz
authors/id/G/GS/GSAR/perl5.00402-bindist03-bc.tar.gz
authors/id/G/GS/GSAR/perl5.00402-bindist04-bc.tar.gz
authors/id/G/GS/GSAR/perl5.00402-bindist04-bc.zip
authors/id/H/HO/HOOO/perl-0.0017.tar.gz
authors/id/K/KR/KRISHPL/perl-5.6-info.tar.gz
authors/id/L/LM/LMOLNAR/perl5.00402-bindist01-dos-djgpp.zip
authors/id/L/LM/LMOLNAR/perl5.00503-bin-1-dos-djgpp.zip
authors/id/M/MS/MSCHWERN/perl-1.0_15.tar.gz
authors/id/R/RC/RCLAMP/perl-1.0_16.tar.gz
authors/id/S/SR/SREZIC/perl-5.005-basicmods-bin-0-arm-linux.tar.gz
authors/id/S/SR/SREZIC/perl-5.005-minimal-bin-0-arm-linux.tar.gz
authors/id/S/SR/SREZIC/perl-5.005-minimal-bin-1-arm-linux.tar.gz
authors/id/S/SR/SREZIC/perl-5.005-Tk-800.023-bin-0-arm-linux.tar.gz
```

These are causing some weird results when compiling lists of `perl` releases.  Unless we hear otherwise, we should mark these as rogue so that they don't get indexed again in future.

**Comment by haarg** [comment]
Currently, the rogue distributions list is only used to prevent things from appearing in search results. It doesn't prevent indexing or having them show in release lists.

Some of these should be handled differently.  For example, most of the items listed in this ticket probably shouldn't be indexed at all.  perl_mlb is similar.  And currently, they aren't indexed because I've deleted them.  Other releases like Bundle-Everything and Acme-DependOnEverything we do want to index, and probably also have show up in search results, but we don't want to appear in dependency lists.

**Comment by mickeyn** [comment]
We can make use of the list in the indexing script to skip them. maybe we should keep them out of the backpan repo altogether by adding an exclusion file to the rsync?

**Comment by haarg** [comment]
I wouldn't want to exclude them from the mirror.  That would only make sense if they were malicious in some way.

**Comment by haarg** [comment]
I've filed #931 to address this.

I looked into the existing things in the rogue distribution lists we have, and many of them are no longer relevant.

  - Acme-DependOnEverything: probably still worth blocking from dependency lists
  - Bundle-Everything: probably still worth blocking from dependency lists
  - [kurila](https://metacpan.org/release/TTY/kurila-1.19_0): no 'latest' release, so not a problem to index normally
  - [perl-5.005_02+apache1.3.3+modperl](https://metacpan.org/release/JBAKER/perl-5.005_02+apache1.3.3+modperl-1.16-bin-bindist1-i386-win32-vc5): Blocked by my PR
  - [perlbench](https://metacpan.org/release/GAAS/perlbench-0.93): Still contains an indexed perlfunc pod file.  Newer unindexed release exists.  Still worth blocking for now, although it might be better if a fixed release was made.
  - [perl_debug](https://metacpan.org/release/GERRIT/perl_debug-5.8.1-2-srctree): no 'latest' release, so not a problem to index normally
  - perl_mlb: Blocked by my PR
  - [pod2texi](https://metacpan.org/release/KRISHPL/pod2texi-0.1): no 'latest' release, so not a problem to index normally
  - [spodcxx](https://metacpan.org/release/OESTERHOL/spodcxx-0.22): no 'latest' release, so not a problem to index normally

**Comment by mickeyn** [comment]
#931 was merged
